containers/src/Kokkos_UnorderedMap.hpp

```diff
@@ -362,14 +403,15 @@ class UnorderedMap {
         bitset_type(Kokkos::Impl::append_to_label(prop_copy, " - bitset"),
                     calculate_capacity(capacity_hint));
 
-    m_hash_lists = size_type_view(
-        Kokkos::Impl::append_to_label(prop_copy_noinit, " - hash list"),
-        Impl::find_hash_size(capacity()));
+    m_hash_lists = allocate_with_property_if_compatible<size_type_view>(
+        Kokkos::Impl::append_to_label(prop_copy, " - hash list"),
+        WithoutInitializing, Impl::find_hash_size(capacity()));
 
-    m_next_index = size_type_view(
-        Kokkos::Impl::append_to_label(prop_copy_noinit, " - next index"),
-        capacity() + 1);  // +1 so that the *_at functions can always return a
-                          // valid reference
+    m_next_index = allocate_with_property_if_compatible<size_type_view>(
```

Comment:
Why would you need sequential host initialization for views that are internal to `UnorderedMap` and are not supposed to contain problematic user types?